apps/angular-demo/angular.json

```diff
@@ -95,7 +95,11 @@
               "browserTarget": "angular-demo:build:production"
             },
             "development": {
-              "browserTarget": "angular-demo:build:development"
+              "browserTarget": "angular-demo:build:development",
+              "headers": {
+                "Cross-Origin-Opener-Policy":"same-origin",
```

Comment:
mostly for #287 browsers require these headers to enable access to `SharedArrayBuffer` which emscripten uses under the hood